### PR TITLE
fixed some wording

### PR DESCRIPTION
**Pull request by xaleeks**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,17 +28,17 @@ Harbor is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
...
+* **Vulnerability Scanning**: Harbor scans images regularly for vulnerabilities and has policy checks to prevent vulnerable images from being deployed.
+* **LDAP/AD support**: Harbor integrates with existing enterprise LDAP/AD for user authentication and management, and supports importing LDAP groups into Harbor that can then be given permissions to specific projects.  
 * **OIDC support**: Harbor leverages OpenID Connect (OIDC) to verify the identity of users authenticated by an external authorization server or identity provider. Single sign-on can be enabled to log into the Harbor portal.  
-* **Image deletion & garbage collection**: Images can be deleted and their space can be recycled.
-* **Notary**: Image authenticity can be ensured.
+* **Image deletion & garbage collection**: Online gc means images can be deleted and their space can be freed up periodically.
```
Comment (reasonerjt) [inline README.md]: What is `online` gc?

File: README.md [hunk 2]
```diff
@@ -28,17 +28,17 @@ Harbor is hosted by the [Cloud Native Computing Foundation](https://cncf.io) (CN
...
+* **Notary**: Images can be signed for authenticiyt and provenance purposes and policies that prevent unsigned images from being deployed can also be activated.
```
Comment (reasonerjt) [inline README.md]: `authenticiyt` is a typo?

File: README.md (hunk 1 above)
Comment (xaleeks) [reply README.md]: good suggestion

File: README.md (hunk 2 above)
Comment (xaleeks) [reply README.md]: sounds good

**Comment by xaleeks** [comment]
fixed errors @reasonerjt

**Pull request merged**